src/js/components/Drop/__tests__/Drop-test.js

```diff
@@ -171,7 +175,7 @@ describe('Drop', () => {
     expectPortal('drop-node').toMatchSnapshot();
   });
 
-  test('props elevation renders', () => {
+  test('elevation prop renders', () => {
```

Comment:
since we're renaming this test, how about just calling it "elevation"? The "renders" suffix is redundant. The "prop" term is also redundant.

Reply:
You don't think that in some cases we'd want the 'prop' to indicate whether it's a component prop or a theme prop?

Reply:
I was thinking we always use the term "property" or "prop" to indicate a React property on a component. I would use the term "theme" for something from the theme. I've been trying to keep test names simpler to make it easier to scan and match with -t. DateInput-test.js shows the naming approach I've been working towards. Worth a discussion.